app/models/orders_package.js

```diff
@@ -33,6 +38,14 @@ export default Model.extend({
     return this.get("designation.code");
   }),
 
+  locationSubscript: Ember.computed("quantity", function() {
+    return Number(this.get("quantity")) >= 0 ? "To" : "From";
+  }),
+
+  arrow: Ember.computed("quantity", function() {
```

Comment:
is this new computed property used somewhere?
Im unable to find it 🤔

Reply:
It is used in hbs, it is added for this orders_packages.